File: Jr/2023/DeliverEDroid.java
// Owen Bélanger-Love

import java.util.Scanner;

public class DeliverEDroid {
    public static void main(String[] args) {
        
        // Create input Scanner
        Scanner input = new Scanner(System.in);

        // Find the amount of packages
        int packages = input.nextInt();

        // Find the amount of collisions
        int collisions = input.nextInt();

        int points = packages * 50 - collisions * 10;

        if (packages > collisions) {
            points += 500;
        }

        // Print result
        System.out.println(points);

        // Close input Scanner
        input.close();
    }
}

File: Jr/2023/DeliverEDroid.c
// Owen Bélanger-Love
#include <stdio.h>

int main() {
    // Initialise variables
    int packages, collisions, points;

    // Get the amount of packages
    scanf("%d", &packages);

    // Get the amount of collisisons
    scanf("%d", &collisions);

    points = packages * 50 - collisions * 10;

    if (packages > collisions) {
        points += 500;
    }
    
    printf("%d",points);
}